src/Tribe/Commerce/PayPal/Main.php

```diff
@@ -2126,7 +2126,7 @@ public function add_cart_url( $urls = [] ) {
 	 * @return array
 	 */
 	public function add_checkout_url( $urls = [] ) {
-		$checkout_url = site_url();
+		$checkout_url = site_url( '/' );
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 6 spaces but found 1 space

<sub>hash: a59c4608292fd1365235cf7495fbd905</sub>